Gemfile

```diff
@@ -118,8 +118,8 @@ gem "strong_migrations"
 gem 'rails-data-migrations'
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem "byebug", platforms: %i[mri mingw x64_mingw]
+  # Adds support for debug
+  gem "debug", "~>1.8"
```

Comment:
**Question** Any reason for locking gem to a particular version ?

Reply:
Not any reason. It's the latest version. 
Will remove if required